common/app/NotFound/Not-Found.jsx

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
-
 // import PropTypes from 'prop-types';
+import Link from '../Router/Link.jsx';
 
 const propTypes = {};
 
 export default function NotFound() {
   return (
-    <div>404 Not Found</div>
+    <div className='container'>
+          <div className='row'>
+            <div className='col-sm'>
+            <div className='alert alert-danger' role='alert'>
+              <h4 className='alert-heading' className='mb-12'>
+                This challenge is not found.</h4>
+                Head back to <Link className='alert-link'
+                   to={'/en/challenges/basic-html-and-html5/' +
```

Comment:
This should link to '/challenges/current-challenge', that way, we are not tying this component down to a single language, or a specific challenge.

Reply:
Gotcha. I tried putting in ' /challenges/current-challenge' but it would not redirect to the home page. However, when I do it in a separate browser, it automatically redirects to /en/challenges/basic-html...

Reply:
To make that work, you need to either use a regular anchor tag or dispatch the currentChallenge action on click.

Reply:
I see. I'll just use a regular anchor tag for the simplicity.